src/ansys/geometry/core/designer/body.py

```diff
@@ -686,6 +697,7 @@ def __init__(
         self._commands_stub = CommandsStub(self._grpc_client.channel)
         self._tessellation = None
         self._fill_style = FillStyle.DEFAULT
+        self._color = None
```

Comment:
Would it be possible to get the information from the server on which color is being used? Even if this only works on 25R1 and onwards. We could implement it as a lazy loaded property (we assign it to None by default and if a user requests the current color, and it is None... we go to the server to retrieve it). And if it is already defined we just return the value. What's your take @umutsoysalansys? :)

Reply:
implemented